src/app/integrations/list/list.component.html

```diff
@@ -1,58 +1,46 @@
 <ipaas-loading [loading]="loading">
   <div class="integrations list">
 
-    <!-- Tabs -->
-    <!--<tabset>-->
-
-    <!-- All -->
-    <!--<tab heading='All'>-->
-    <div class="integration col-xs-12"
-         *ngFor="let integration of integrations">
+    <!-- Toast Notification Container -->
+    <toaster-container></toaster-container>
```

Comment:
Why remove the toast notification container? Also, we need those tabs and tabsets for implementing the most recent designs.